app/src/main/java/ar/com/wolox/android/example/ui/auth/login/AuthPresenter.kt

```diff
@@ -10,10 +15,16 @@ import javax.inject.Inject
 
 class AuthPresenter @Inject constructor(
     private val loginRepository: LoginRepository,
+    private val context: Context,
     private val userSession: UserSession
 ) : CoroutineBasePresenter<AuthView>() {
 
+    @RequiresApi(Build.VERSION_CODES.M)
```

Comment:
Instead of doing this, you can also do: `if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) { doSomething() }`